_includes/header.html

```diff
@@ -4,7 +4,7 @@
     {% assign default_paths = site.pages | map: "path" %}
     {% assign page_paths = site.header_pages | default: default_paths %}
 
-    <a href="https://www.mechanicalrock.io" class="logo-holder">
+    <a href="https://www.mechanicalrock.github.io" class="logo-holder">
```

Comment:
This should redirect to our main homepage - not the blog homepage.

Reply:
I had a discussion with Lucy about this. If we leave this as is, once we enter a search the only way to get back to the blog home page would be by clicking on the clear button. Are we okay with this?
![image](https://user-images.githubusercontent.com/7382785/114807888-cbf18800-9dd9-11eb-9ecb-8d22dbbe315b.png)

Reply:
Interesting.  

I see the difference in domains as much less an issue - since the blog is just an extension of the website experience IMO.

* The link from our main website needs a separate discussion (out of scope of this PR)
* Styling of the header and navigation, etc for consistency (related to design system) is a good discussion to have, but again outside the scope of this PR.

> The user has landed to this content...

Yes, but if you consider the website / blogs as an extension of the SAME content, then the lines are blurred (again the domain is a furphy IMO).  Need data on this and experimentation to make changes here, but this is definitely outside the scope of this PR!

We already have links within each post to the main site as well.

For the purposes of this PR - please revert so it re-directs to the main website.  Moving the search box into the header resolves this issues around where results should go.

Reply:
FYI @Hamo69

Reply:
Yep, it's exactly what Q said: "the user has landed to this content by any means necessary and to redirect them away from the domain to our main website feels jarring."
It does and it is frigging annoying